customTests/azure_cpu_stream.nhc

```diff
@@ -21,26 +40,24 @@ function check_cpu_stream(){
     SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
     SKU=$(echo "$SKU" | tr '[:upper:]' '[:lower:]')
     
-    
-
-
-    if echo $SKU | grep "hx176rs" || echo $SKU | grep "hb176rs"; then
-        stream=`OMP_SCHEDULE=static && OMP_DYNAMIC=false && OMP_MAX_ACTIVE_LEVELS=1 && OMP_STACKSIZE=256M OMP_PROC_BIND=true && OMP_NUM_THREADS=24 && 
-            OMP_PLACES='{0:1},{8:1},{16:1},{24:1},{32:1},{38:1},{44:1},{52:1},{60:1},{68:1},{76:1},{82:1},{88:1},{96:1},{104:1},{112:1},{120:1},{126:1},{132:1},{140:1},{148:1},{156:1},{164:1},{170:1}' \
-            $EXE_DIR/stream.exe |grep -A 4 'Copy:' `
+    if echo $SKU | grep "hx176" || echo $SKU | grep "hb176"; then
```

Comment:
hmmm yea my reasoning for this was because I want the other H176 sizes to go into this condition and same with the other comment you have. I see how this can be confusing though. Maybe I can make a list of HB/HX list and if the sku is in the list it will meet the condition

Reply:
addressed for both